src/main/java/uk/gov/hmcts/ccd/domain/model/definition/CaseFieldDefinition.java

```diff
@@ -3,16 +3,15 @@
 import com.fasterxml.jackson.annotation.JsonIgnore;
 import com.fasterxml.jackson.annotation.JsonProperty;
 import io.swagger.annotations.ApiModel;
-import lombok.ToString;
-import uk.gov.hmcts.ccd.domain.model.aggregated.CommonField;
```

Comment:
Could you please move  the imports and take developed  version ?

Reply:
reverted to develop version